partcad/src/partcad/shape_ai.py

```
@@ -51,6 +51,8 @@ async def get_summary_async(self, project=None):
 
         config = {
             "model": ("gpt-4o" if not user_config.openai_api_key is None else "gemini-1.5-pro"),
+            # TODO: ERROR: Failed to associate the model gemini-1.5-pro with the provider
+            "provider": ("openai" if not user_config.openai_api_key is None else "google"),
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)